codec/src/main/java/io/netty/handler/codec/DefaultTextHeaders.java

```diff
@@ -579,7 +428,8 @@ private TextHeaders addEscapedValue(CharSequence name, CharSequence escapedValue
         }
 
         private <T> CharSequence commaSeparate(CsvValueEscaper<T> escaper, T... values) {
-            StringBuilder sb = new StringBuilder(values.length * DEFAULT_VALUE_SIZE);
```

Comment:
@Scottmitch 
> I don't see how we are decreasing complexity here.

by deleting it and not having anyone worry about it.

> If it is upfront perhaps it is easier to notice, or potentially easier to configure.

who is going to configure it? It was a private static final that is used once and without any meaning really. it's just some randomly chosen value to size a string nobody cares about :-).